src/protocol/prf_sharding/mod.rs

```diff
@@ -68,30 +97,37 @@ impl InputsRequiredFromPrevRow {
     ///         - `did_trigger_get_attributed` - a secret-shared bit indicating if this row corresponds to a trigger event
     ///           which was attributed. Might be able to reveal this (after a shuffle and the addition of dummies) to minimize
     ///           the amount of processing work that must be done in the Aggregation stage.
-    pub async fn compute_row_with_previous<C, BK, TV>(
+    #[allow(clippy::too_many_lines)]
```

Comment:
Does it still exceed the max number of lines now, after this latest refactoring?

Reply:
by 2 lines :(